CHANGELOG.md

```diff
@@ -2,10 +2,12 @@
 
 ## Next Release
 
+## 2022-11-28-8523
+
 ### Internal
 
 - Bump Faker to v1.2.9
-- Bump sqlite-android version to 3.39.2
+- Bump sqlite-android version to 3.39.
```

Comment:
The `sqlite-android version` given is `3.39.` instead of `3.39.2`

Reply:
Good catch